tests/acceptance/tooltip-auto-close-test.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+
+var application;
+
+module('Acceptance | tooltip auto close', {
+  beforeEach: function() {
+    application = startApp();
+  },
+
+  afterEach: function() {
+    Ember.run(application, 'destroy');
+  }
+});
+
+test('Auto-closing tooltips using duration parameter', function(assert) {
+  visit('/tooltip-auto-close');
+
+  assert.expect(25);
+
+  assertTooltipProperties(assert, 'auto-close-basic', {
```

Comment:
Is this testing the effect of the `duration` property or just the initial tooltip render?